lib/HLSL/DxilSignature.cpp

```diff
@@ -53,7 +54,7 @@ bool DxilSignature::IsOutput() const {
 }
 
 unique_ptr<DxilSignatureElement> DxilSignature::CreateElement() {
-  return unique_ptr<DxilSignatureElement>(new DxilSignatureElement(m_sigPointKind));
+  return llvm::make_unique<DxilSignatureElement>(m_sigPointKind);
```

Comment:
I introduced a bunch more of these recently, I'll scan and update the new ones after this is merged. Very useful to avoid leaks under exceptions :smile: